src/components/SubHeader/index.tsx

```diff
@@ -3,26 +3,34 @@ import { palette } from "@leafygreen-ui/palette";
 import Icon from "components/Icon";
 import { StyledLink } from "components/styles";
 import { size, subheaderHeight } from "constants/tokens";
+import { useLogContext } from "context/LogContext";
 
 const { gray } = palette;
 
 /** TODO: EVG-17534 */
 interface SubHeaderProps {
   isUploadedLog: boolean;
 }
-const SubHeader: React.FC<SubHeaderProps> = ({ isUploadedLog }) => (
-  <Container>
-    <StyledIcon glyph="EvergreenLogo" />
-    {isUploadedLog ? (
-      <span>Thanks for uploading</span>
-    ) : (
-      <StyledLink href="/test">Task Page</StyledLink>
-    )}
-  </Container>
-);
+const SubHeader: React.FC<SubHeaderProps> = ({ isUploadedLog }) => {
+  const { fileName } = useLogContext();
 
-const StyledIcon = styled(Icon)`
-  padding-left: ${size.l};
+  return (
+    <Container>
+      {isUploadedLog ? (
+        <span>File name: ./{fileName}</span>
```

Comment:
[nit] I would probably omit the `./` here, feel like it implies some type of filesystem/hierarchy where there is none. And developers can be finicky 😁